Kor Efret jobs and missions (#5219)
endless-sky/endless-sky: data/korath/korath jobs.txt

	on complete
		payment
		dialog phrase "generic cargo delivery payment"
		
mission "Kor Efreti: Cargo [1]"
	name "Delivery to <planet>"
	job
	repeat
	description "Deliver <cargo> to <destination>. Payment is <payment>."
	cargo random 10 2 .14
	to offer
		has "Translator Development 4: done"
		random < 20
	source
		government "Kor Efret"
	destination
		distance 1 8
		government "Kor Efret" "Wanderer"
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		payment
		payment 500
		dialog phrase "generic cargo delivery payment"		

mission "Kor Efreti: Food Cargo [0]"
	name "Delivery to <planet>"
	job
	repeat
	description "Deliver <cargo> to <destination>. Payment is <payment>."
	cargo "Food" 5 3 .16
	to offer
		has "Translator Development 4: done"
	source
		government "Kor Efret"
		not attributes "station"
	destination
		distance 1 8
		government "Kor Efret"
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		payment
		dialog phrase "generic cargo delivery payment"

mission "Kor Efreti: Food Cargo [1]"
	name "Delivery to <planet>"
	job
	repeat
	description "Deliver <cargo> to <destination>. Payment is <payment>."
	cargo "Food" 5 3 .14
	to offer

> 		random < 70
> 		has "Translator Development 4: done"

Reviewer: Consistency change.
```suggestion
		has "Translator Development 4: done"
		random < 70
```

	on visit
		dialog phrase "generic passenger on visit"
	on complete
		payment
		dialog phrase "generic passenger dropoff payment"

mission "Kor Efreti: Passengers [1]"
	name "Korath passengers to <planet>"
	job
	repeat
	description "Bring this group of <bunks> Kor Efreti to <destination>. Payment is <payment>."
	passengers 3 2 .1
	to offer
		has "Wanderers: Sestor: Final: done"
		random < 60
	source
		government "Kor Efret"
		not attributes "station"
	destination
		distance 1 8
		government "Kor Efret"
	on visit
		dialog phrase "generic passenger on visit"
	on complete
		payment
		payment 500
		dialog phrase "generic passenger dropoff payment"

mission "Kor Efreti: Settlers"
	name "Korath settlers to <planet>"
	job
	repeat
	description "Take these <bunks> migrating Kor Efreti and their <cargo> to the new Wanderer-Korath settlement on <destination>. Payment is <payment>."
	passengers 10 5 .15
	cargo "supplies" 5 3 .16
	to offer
		has "Wanderers: Sestor: Final: done"
		random < 90
	source
		government "Kor Efret"
		not attributes "station"
	destination "Sabira Eseskrai"
	on visit
		dialog phrase "generic cargo and passenger on visit"
	on complete
		payment
		payment 2000

> 		dialog "While the settlers depart your ship, the Korath in charge hands you <payment>."

Reviewer: ```suggestion
		dialog "The Korath settlers slowly depart your ship with their cargo, seemingly apprehensive to step onto this once-dead world. The Korath in charge gives you a gesture of thanks and hands you <payment>."
```

mission "Kor Efreti to Wanderers: Farming"

> 	name "Learning Wanderer farming methods"

Reviewer: The Korath are learning. The player is helping with the teaching.
```suggestion
	name "Teaching Wanderer farming methods"
```

	job
	repeat

> 	description "Take <bunks> Kor Efreti to learn Wanderer farming methods on <stopovers>, and return them to <destination>. Payment is <payment>."

Reviewer: ```suggestion
	description "Take <bunks> Kor Efreti to learn Wanderer farming methods on <stopovers>, then return them to <planet>. Payment is <payment>."
```
